upcoming-release-notes/1419.md

```diff
@@ -0,0 +1,6 @@
+---
+category: Maintenance
+authors: [MatissJanis]
+---
+
+Moving away from barrell `common` imports to more specific per-component imports (part 2)
```

Comment:
Sorry, still looks off. Assuming you meant “barrel”
```suggestion
Moving away from barrel `common` imports to more specific per-component imports (part 2)
```

Reply:
Why is my spellchecker not working for this word :D